model_analyzer/perf_analyzer/perf_config.py

```diff
@@ -71,6 +72,7 @@ class PerfAnalyzerConfig:
         "metrics-url",
         "metrics-interval",
         "bls-composing-models",
+        "max-token-count",
```

Comment:
Should be `--request-parameter=max_token:<n>:int`

Reply:
"max_token:<n>:int" is a string

Reply:
I've made the change.